### ISSUE TITLE
LED strap operation is not possible.

### ISSUE THREAD

**Issue by tdoypwls** [onlaj/Piano-LED-Visualizer#360]
I followed the guide exactly and confirmed that the LCD booted normally.
However, the LED strap only blinks and cannot be control at all.

All operations such as MIDI playback, piano playing, and LED color change are impossible.

The LED strap I bought is 144leds, IP67, 2m.
I connected the LED strap wire to Gnd(-) and Gpio15(control) of Raspberry Pi PCB correctly.

Help plz...

LED strap link : opt(144leds, IP67, 2m.) // 
https://a.aliexpress.com/_mOZELuA

![KakaoTalk_20220608_153404114](https://user-images.githubusercontent.com/107099691/172548151-04315db3-a1cf-4089-a956-97587d2f771a.jpg)
![KakaoTalk_20220608_153404114_01](https://user-images.githubusercontent.com/107099691/172548198-217429df-48e2-4924-a8b1-cd49de2809c5.jpg)

https://user-images.githubusercontent.com/107099691/172548997-041fcc67-02a6-4af2-85d0-bd7f4dc7b7b5.mp4

![화면 캡처 2022-06-08 155056](https://user-images.githubusercontent.com/107099691/172550954-294c4075-eb12-4776-8df6-e0c0fa975fcb.png)

**Comment by onlaj** [comment]
Hello.
It's really hard to tell from the photos you posted, but are you sure the wiring is correct? It seems like data cable is going to the wrong hole in the socket.

**Comment by tdoypwls** [comment]
Yes. It didn't work, so I tested all sockets. I also tried the opposite.

as in the video
Connected correctly to GND and GPIO 15 of Raspberry Pi.

GND is connected to the 3rd port from the left, and the LED data GPIO is connected to the 6th port from the left.

The LED does not work even if I reverse it to the right just in case.

**Comment by onlaj** [comment]
I meant the other side of cables. Does it go to the correct wires on led strip? From the pictures it looks like data (from RPI) is connected to (+) (from the led strip).

**Comment by tdoypwls** [comment]
Yes, there are 3 cables attached to the LED strip.

The order is + / data / - .

+ is the red cable

data is a black cable

- is the white cable.


- is connected to GND of Raspberry Pi board and - of 5v,

+ is only connected to 5v +.

data is connected to GPIO 15 with black and yellow wires.

**Comment by tdoypwls** [comment]
The LED strap I have is

The order is different from the last photo example image.

my strap is
(+) / data (-) and

example image is
(data) blue
(+) red
(-) It is in black

**Comment by onlaj** [comment]
> The order is different from the last photo example image.

Yeah, sorry, it was little missleading. I checked my led strip and data is also in the middle.

Just to make sure it's not a problem with my code you can try to run [strandtest.py](https://github.com/onlaj/Piano-LED-Visualizer/blob/master/tests/strandtest.py) . If it still doesn't work I suspect there is something wrong with either RPI gpio pins or led strip.

**Comment by tdoypwls** [comment]
Thank you very much.

Let's try it again.

If it doesn't work, I guess I'll have to buy the strip again...

**Comment by tdoypwls** [comment]
I ran strandtest.py. However, No. The module named rpi_ws281x is printed.
I did the following:

cd ~
sudo pip uninstall rpi_ws281x
sudo pip install rpi_ws281x
sudo pip3 uninstall rpi_ws281x
sudo pip3 install rpi_ws281x

But it keeps saying that the module does not exist, what should I do
![화면 캡처 2022-06-09 082524](https://user-images.githubusercontent.com/107099691/172733752-2b55d8d8-d03d-4aeb-9fd2-5a7a9bb02c7c.png)
?
